### PR TITLE
Ukrainian and Russian translations of assembly instructions

### PR DESCRIPTION
**Pull request by DenysGonchar**
This PR introduces the following changes:

- Significant rework of the Ukrainian translation. Previous version was really hard to read, it seems that it's done using machine translation. Also it required the same fix as Polish translation, see [issue#19](https://github.com/GliaX/tourniquet/issues/19)
- Adding a Russian translation (see [issue#6](https://github.com/GliaX/tourniquet/issues/6))
- Adding illustrations for the sewing process. This help to avoid any misunderstanding of the description in a text form.
- Change of the overlap of Velcro pieces from 5 mm to 1 cm. Sewing is hard with 5 mm overlap, especially sewing of loop piece on top of the hook.

UPDATE:

English version has been updated since I started the work on this branch. I have made the corresponding changes to the UA/PL/RU translations. Also I reworded a bit the Polish translation, and made some minor changes to the English version.

The Arabic and French translations must be reviewed and adapted as well, but I cannot do it by my own.

### CODE REVIEW

**Comment by tareko** [comment]
Thank you so much for this! @jo-majs , could you do a review of the revised Polish instructions as well? I'll add if so.

Thank you.

**Comment by DenysGonchar** [comment]
I have noticed that English version of the manual updated information about stitching of the velcro parts. I'll upload the improved illustrations later today and add the corresponding changes in the UA/PL/RU translations (just a few sentences to change).

**Comment by tareko** [comment]
Yes, correct @DenysGonchar . Thank you for this! Let me know and I'll make the commit from this PR when you're ready.

**Comment by DenysGonchar** [comment]
@tareko I think it is ready for a proofread. I have also updated PR's description.
@defo89 please check the Russian translation

File: assembly_instructions/INSTRUCTIONS_RU.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Список материалов
+* Черная лента-липучка шириной 3,5-3,85 см для лицевой стороны ремня, состоит из жесткой (hook)  и мягкой (loop) частей.
+* Белая лента-липучка шириной 2 см для клипсы: жесткая и мягкая части.
+* Хлопково-полиэстровая тесьма шириной 3,5-3,85 см и толщиной примерно 1,5 мм для тыльной стороны ремня.
```
Comment (defo89) [inline assembly_instructions/INSTRUCTIONS_RU.md:4]: Хлопко-полиэстеровая

**Comment by defo89** [comment]
@DenysGonchar apart from one small type, Russian translation lgtm. Thanks for your contribution!

**Comment by jenwilsonglia** [comment]
Made another pull request with these changes, except for the EN instructions, which will be updated asap. (EN instructions were conflicting with updates that had already been merged.)

**Comment by DenysGonchar** [comment]
@jenwilsonglia if you want to build any community around the project, you should not do it this way. Authorship is the only "reward" for a committer. Copy-pasting other's changes and pushing it as your own eliminates even this. I could easily update this PR if you ask about it, but you gave me no chance to do it.

**Comment by jenwilsonglia** [comment]
[Denys](https://github.com/DenysGonchar) please accept my apology as I'm new to Github and was just trying to clean up the repository. I wasn't aware I was offending anyone, but I'm just learning all about open source, so thanks for letting me know I made a big mistake. I would like to retract my changes and reopen your pull request, but with the request that the English instructions are not changed, since they've been significantly updated since your pull request. I would, however, like to incorporate your graphic images into the updated English instructions. Please let me know if that's ok with you, or if you have any other ideas as to how I can help rectify this.

**Comment by DenysGonchar** [comment]
> [Denys](https://github.com/DenysGonchar) please accept my apology as I'm new to Github and was just trying to clean up the repository. I wasn't aware I was offending anyone, but I'm just learning all about open source, so thanks for letting me know I made a big mistake. I would like to retract my changes and reopen your pull request, but with the request that the English instructions are not changed, since they've been significantly updated since your pull request. I would, however, like to incorporate your graphic images into the updated English instructions. Please let me know if that's ok with you, or if you have any other ideas as to how I can help rectify this.

Please don't retract anything, that would create even more mess. There is a golden rule of git (and other version control systems) no-one should ever rewrite the main (`master`) branch, changes must be done only in incremental way. That's because other participants use main branch as a starting point for their changes. 

I know that you are learning, and it's not an issue for me personally. However, other committers may treat it badly.

Just try to contact PR authors first, and if they are unresponsive during significant time - feel free to act by yourself. Also, you can just take original commits of other people instead of reapplying their changes, and put your own changes on top. That probably would be the most correct way. If you experience any difficulties with git, I'm happy to help with it.

**Comment by jenwilsonglia** [comment]
The conflict is with the EN instructions, which were substantially updated after this pull request. Can I please update the EN instructions with your graphics? And can I have you not include the EN instructions on this pull request so that I can push it asap.

**Comment by jenwilsonglia** [comment]
Ok Denys,

But, um, I already retracted it. So, I don't think it's that big of a mess, but it means I just need to get your pull request approved. Thanks for your help so far. There's a lot to learn! But, hey, that's what I'm doing by doing all this...I guess:)

-Jen

March 14, 2023 10:47 AM, "Denys Gonchar" ***@***.*** ***@***.******@***.***>)> wrote:
	Denys (https://github.com/DenysGonchar) please accept my apology as I'm new to Github and was just trying to clean up the repository. I wasn't aware I was offending anyone, but I'm just learning all about open source, so thanks for letting me know I made a big mistake. I would like to retract my changes and reopen your pull request, but with the request that the English instructions are not changed, since they've been significantly updated since your pull request. I would, however, like to incorporate your graphic images into the updated English instructions. Please let me know if that's ok with you, or if you have any other ideas as to how I can help rectify this.

	Please don't retract anything, that would create even more mess. There is a golden rule of git (and other version control systems) no-one should ever rewrite the main (master) branch, changes must be done only in incremental way. That's because other participants use main branch as a starting point for their changes.

	I know that you are learning, and it's not an issue for me personally. However, other committers may treat it badly.

	Just try to contact PR authors first, and if they are unresponsive during significant time - feel free to act by yourself. Also, you can just take original commits of other people instead of reapplying their changes, and put your own changes on top. That probably would be the most correct way. If you experience any difficulties with git, I'm happy to help with it.

**Comment by DenysGonchar** [comment]
`But, um, I already retracted it.`  that was fast :)
please give me 15 minutes, I will make an update to this PR.

**Comment by jenwilsonglia** [comment]
Yeah, I saw how to do it and wanted to also just see how this all works. Anyway, I think it's the best way to do this right now that the repository is fairly quiet and I'm trying to learn about it so I can maintain it better.
March 14, 2023 10:54 AM, "Denys Gonchar" ***@***.*** ***@***.******@***.***>)> wrote:
	But, um, I already retracted it. that was fast :)
please give me 15 minutes, I will make an update to this PR.

**Comment by DenysGonchar** [comment]
It should be fine now.

**Comment by jenwilsonglia** [comment]
Ok, thanks Denys! I will get this done today.

-Jen

March 14, 2023 11:10 AM, "Denys Gonchar" ***@***.*** ***@***.******@***.***>)> wrote:
	It should be fine now.

**Comment by jenwilsonglia** [comment]
I will be adding these illustrations to the updated EN instructions.

**Comment by DenysGonchar** [comment]
Sure, let me know if any of them must be updated. BTW, there are original illustration files in Adobe Illustrator format. It's much easier to modify them. But it's proprietary format, so I'm not sure if you want to add them in this repo.

**Comment by jenwilsonglia** [comment]
Thanks Denys, I'll let you know. I think we're good without the .ai files, but if we need them to modify anything, I'll also let you know.

March 14, 2023 12:33 PM, "Denys Gonchar" ***@***.*** ***@***.******@***.***>)> wrote:
	Sure, let me know if any of them must be updated. BTW, there are original illustration files in Adobe Illustrator format. It's much easier to modify them. But it's proprietary format, so I'm not sure if you want to add them in this repo.

**Pull request merged**